libs/styles/src/lib/scss/components/_notice.scss

```diff
@@ -0,0 +1,58 @@
+@mixin cvi-notice($base: &) {
+  width: 100%;
+  #{$base}__container {
+    border-radius: var(--cvi-radius-s);
+    display: flex;
+    color: get-color(black-coral-12);
+    fill: get-color(black-coral-12);
+    font-size: 14px;
+  }
+
+  #{$base}__content {
+    align-items: center;
+    padding: var(--cvi-spacing-valga);
+    width: 100%;
+  }
+
+  #{$base}__icon-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 10px;
+  }
+
+  #{$base}--severity-success {
+    border: 1px solid var(--cvi-color-black-coral-2);
+
+    & #{$base}__icon-container {
+      background-color: var(--cvi-color-sea-green-2);
+      fill: var(--cvi-color-sea-green-10);
+    }
+  }
+
+  #{$base}--severity-info {
+    border: 1px solid var(--cvi-color-black-coral-2);
+    & #{$base}__icon-container {
+      background-color: var(--cvi-color-sapphire-blue-2);
+      fill: var(--cvi-color-sapphire-blue-10);
+    }
+  }
+
+  #{$base}--severity-error {
+    border: 1px solid var(--cvi-color-jasper-10);
+
+    & #{$base}__icon-container {
+      background-color: var(--cvi-color-jasper-2);
+      fill: var(--cvi-color-jasper-10);
+    }
+  }
+
+  #{$base}--dark-title {
```

Comment:
According to [BEM rules](https://getbem.com/naming/), `--` indicates a modifier class. This class amends some styling already set by a different class on the same DOM node. 

In this case, it probably should be an element class – `#{$base}__title`. 

The "dark" subword can be omitted because there's only one kind of title in this component, and making the name more specific might confuse people (if this is the dark one, there should be a light one somewhere?).